go/libkb/merkle_client.go

```diff
@@ -685,13 +688,17 @@ func (mc *MerkleClient) LastRoot() *MerkleRoot {
 }
 
 func (mc *MerkleClient) FirstSeqnoWithSkips() *Seqno {
+
 	if mc.G().Env.GetRunMode() == ProductionRunMode {
-		return FirstProdMerkleSeqnoWithSkips
+		if mc.G().Env.GetFeatureFlags().Admin() {
+			return &FirstProdMerkleSeqnoWithSkips
```

Comment:
What should happen if I'm in prod mode but I set `--server=http://localhost:3000` or something like that? Is this a configuration that was working before but that won't work after this?